libvmi/libvmi.h

```diff
@@ -1731,6 +1731,34 @@ uint64_t vmi_get_offset(
     const char *offset_name);
 
 /**
+ * Get the memory offset associated with the given symbol and subsymbol in the struct
+ * @param[in] vmi LibVMI instance
+ * @param[in] struct_name String name for desired symbol
+ * @param[in] subsymbol String name for desired subsymbol
+ * @param[out] offset of subsymbol in struct (symbol)
+ *
+ * @return VMI_SUCCESS or VMI_FAILURE
+ */
+status_t vmi_get_kernel_struct_offset(
+    vmi_instance_t vmi,
+    const char* struct_name,
+    const char* subsymbol,
+    addr_t *rva);
```

Comment:
Parameters named subsymbol and rva seem strange in the context of this function. I understand that they originate at a Rekall layer, but for public exposure in the LibVMI API, please keep them as generic as possible.